crate/operator/handlers/handle_update_allowed_cidrs.py

```diff
@@ -31,6 +34,15 @@ async def update_service_allowed_cidrs(
     change: DiffItem = diff[0]
     logger.info(f"Updating load balancer source ranges to {change.new}")
 
+    await send_operation_progress_notification(
+        namespace=namespace,
+        name=name,
+        message="Waiting for IP Network Whitelist to be updated.",
```

Comment:
Not really waiting for anything? :) 

```suggestion
        message="Updating IP Network Whitelist",
```